nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -2492,4 +2492,32 @@ public void defaultLibraryModelsObjectNonNull() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void checkForNullSupport() {
+    compilationHelper
+        .addSourceLines(
+            "Nullable.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "class TestNullable {",
+            "  @Nullable",
+            "  Object nullable = new Object();",
+            "  public void setNullable(@Nullable Object nullable) {this.nullable = nullable;}",
+            "  // BUG: Diagnostic contains: dereferenced expression nullable is @Nullable",
+            "  public void run() {System.out.println(nullable.toString());}",
+            "}")
+        .addSourceLines(
+            "CheckForNull.java",
```

Comment:
This should be `TestCheckForNull.java`, right?

Reply:
Yes, I'll fix that. Is this value actually used somewhere?